### PR TITLE
Docs: Add Property Types

### PR DESCRIPTION
**Pull request by jaruba**

### CODE REVIEW

File: docs/api/responses/meta.md [hunk 1]
```diff
@@ -2,72 +2,72 @@
...
+``description`` - _optional_ - string, a few sentances describing your content
 
-``releaseInfo`` - _optional_ - string - year the content came out ; if it's ``series`` or ``channel``, use a start and end years split by a tide - e.g. ``"2000-2014"``. If it's still running, use a format like ``"2000-"``
+``releaseInfo`` - _optional_ - string, year the content came out ; if it's ``series`` or ``channel``, use a start and end years split by a tide - e.g. ``"2000-2014"``. If it's still running, use a format like ``"2000-"``
 
-``director``, ``cast`` - _optional_  - directors and cast, both arrays of names
+``director``, ``cast`` - _optional_  - directors and cast, both arrays of names (string)
 
-``imdbRating`` -  _optional_ - IMDb rating, a number from 0 to 10 ; use if applicable
+``imdbRating`` -  _optional_ - string, IMDb rating, a number from 0.0 to 10.0 ; use if applicable
```
Comment (Ivshti) [inline docs/api/responses/meta.md]: isn't this a number?

File: docs/api/responses/meta.md [hunk 2]
```diff
@@ -2,72 +2,72 @@
...
 
-``dvdRelease`` - _optional_ - DVD release date
+``dvdRelease`` - _optional_ - string, DVD release date
 
-``released`` - _optional_ - initial release date; for movies, this is the cinema debut
+``released`` - _optional_ - string, initial release date; for movies, this is the cinema debut
```
Comment (Ivshti) [inline docs/api/responses/meta.md]: this should be an ISO 8601 date perhaps?

File: docs/api/responses/meta.md [hunk 3]
```diff
@@ -2,72 +2,72 @@
...
 #### Video object
 
-``id`` - **required** - ID of the video
+``id`` - **required** - string, ID of the video
 
-``title`` - **required** - title of the video
+``title`` - **required** - string, title of the video
 
-``released`` - **required** - Date, publish date of the video; for episodes, this should be the initial air date
+``released`` - **required** - string, publish date of the video; for episodes, this should be the initial air date, e.g. "2010-12-06T05:00:00.000Z"
```
Comment (Ivshti) [inline docs/api/responses/meta.md]: again, yes, this is a string but should be in an iso date format

File: docs/api/responses/stream.md [hunk 4]
```diff
@@ -4,16 +4,16 @@ Used as a response for [`defineStreamHandler`](../requests/defineStreamHandler.m
 
 **One of the following must be passed** to point to the stream itself
 
-* ``url`` - direct URL to a video stream - http, https, rtmp protocols supported
-* ``ytId`` - youtube video ID, plays using the built-in YouTube player
-* ``infoHash`` and/or ``fileIdx`` - info hash of a torrent file, and mapIdx is the index of the video file within the torrent; **if fileIdx is not specified, the largest file in the torrent will be selected**
-* ``externalUrl`` - URL to the video, which should be opened in a browser (webpage), e.g. link to Netflix
+* ``url`` - string, direct URL to a video stream - http, https, rtmp protocols supported
+* ``ytId`` - string, youtube video ID, plays using the built-in YouTube player
+* ``infoHash`` and/or ``fileIdx`` - string, info hash of a torrent file, and mapIdx is the index of the video file within the torrent; **if fileIdx is not specified, the largest file in the torrent will be selected**
```
Comment (Ivshti) [inline docs/api/responses/stream.md]: file idx is a number, integer

File: docs/api/responses/meta.md (hunk 1 above)
Comment (jaruba) [reply docs/api/responses/meta.md]: i checked a meta response from cinemeta before writing "string", cinemeta returns them as strings

**Pull request merged**